### PR TITLE
Calculate CRC lookup table at compile time

### PR DESCRIPTION
**Pull request by feliwir**
The table is still identical to the table before

### CODE REVIEW

File: src/game/common/array.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+
+#ifndef ARRAY_H
```
Comment (janisozaur) [inline src/game/common/array.h:17]: superfluous in presence of `#pragma once` just above?

**Comment by feliwir** [comment]
@janisozaur
i know, but the other headers have it aswell

**Comment by feliwir** [comment]
Seems to be some style. Maybe @Omniblade can comment on this

**Comment by OmniBlade** [comment]
Originally I wasn't sure what compilers and level of C++ I was intending to support and pragma once isn't actually standard although pretty much all C++11 compilers do support it so I've been hedging my bets as it were and couldn't face changing the style after the fact.

**Comment by feliwir** [comment]
i don't know of any compiler that doesn't have #pragma once these days

**Comment by OmniBlade** [comment]
Superceeded by #30

**Pull request closed without merging**